agent/src/main/java/com/microsoft/hydralab/agent/runner/TestRunner.java

```diff
@@ -163,7 +163,11 @@ protected void setUp(DeviceInfo deviceInfo, TestTask testTask, TestRun testRun)
     protected void tearDown(DeviceInfo deviceInfo, TestTask testTask, TestRun testRun) {
         // stop performance test
         if (performanceTestManagementService != null) {
-            performanceTestManagementService.testTearDown(deviceInfo, log);
+            try {
```

Comment:
nit: if performance test manager service has exception internal, we should handle the error inside the service itself instead of throw all the exception to the host service(runner)

Reply:
Thanks, will do in the future